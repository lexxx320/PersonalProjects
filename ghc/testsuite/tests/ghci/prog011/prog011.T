test('prog011', reqlib('mtl'), ghci_script, ['prog011.script'])

