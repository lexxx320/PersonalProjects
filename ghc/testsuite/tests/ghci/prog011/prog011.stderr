
prog011.hx:14:22: Empty 'do' block
